client/src/i18n/stored-locale.ts

```diff
@@ -0,0 +1,11 @@
+import { Locale } from "./i18n-config";
+
+const K_LOCALE = "locale";
+
+export function getStoredLocale() {
```

Comment:
I may have missed it, but are we calling this function anywhere else?

Reply:
Oops, I left it out--it's supposed to be called in `getBestMatchLocale()` when matching user/browser and app locales--if there is a stored locale, we should use it. Good catch!